src/shared/idriss/components/send/container.tsx

```diff
@@ -80,6 +82,7 @@ export const Container = memo(
         event.preventDefault();
         setIsVisible(true);
         updatePosition();
+        onOpen?.();
```

Comment:
Oh, I've never seen that way of conditional function invocation, and I like it 👌👌